components/avatar/index.css

```diff
@@ -10,107 +10,126 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+}
+
+@media (forced-colors: active) {
+  .spectrum-Avatar {
+    --highcontrast-avatar-color-opacity-disabled: 1;
+    --highcontrast-avatar-focus-indicator-color: Highlight;
   }
 }
 
 .spectrum-Avatar {
   display: inline-block;
   position: relative;
-  inline-size: var(--spectrum-avatar-width);
-  block-size: var(--spectrum-avatar-height);
+  inline-size: var(--mod-avatar-inline-size, var(--spectrum-avatar-inline-size));
+  block-size: var(--mod-avatar-block-size, var(--spectrum-avatar-block-size));
 
-  border-radius: var(--spectrum-avatar-border-radius);
+  border-radius: var(--mod-avatar-border-radius, var(--spectrum-avatar-border-radius));
   border-width: 0;
 
+  outline: none;
+
   -webkit-user-drag: none;
   user-select: none;
-  overflow: hidden;
 
-  &:after {
-    content: '';
+  overflow: visible;
+
+  opacity: var(--mod-avatar-color-opacity, var(--spectrum-avatar-color-opacity));
+
+  &.is-disabled {
+    opacity: var(--highcontrast-avatar-color-opacity-disabled, var(--mod-avatar-color-opacity-disabled, var(--spectrum-avatar-color-opacity-disabled)));
+  }
+
+  &:focus-within:not(.is-disabled) .spectrum-Avatar-link::after {
+    pointer-events: none;
+    content: "";
     position: absolute;
-    inset-block-start: 0;
-    inset-block-end: 0;
-    inset-inline-start: 0;
-    inset-inline-end: 0;
-    box-sizing: border-box;
-    border-width: var(--spectrum-avatar-border-size);
+    top: calc(
+      var(--mod-avatar-focus-indicator-gap, var(--spectrum-avatar-focus-indicator-gap)) * -2);
+    left: calc(
+      var(--mod-avatar-focus-indicator-gap, var(--spectrum-avatar-focus-indicator-gap)) * -2);
+  
+    inline-size: calc(
+      var(--mod-avatar-inline-size, var(--spectrum-avatar-inline-size)) +
+      calc(var(--mod-avatar-focus-indicator-gap, var(--spectrum-avatar-focus-indicator-gap)) * 2));
+    block-size: calc(
+      var(--mod-avatar-inline-size, var(--spectrum-avatar-inline-size)) +
+      calc(var(--mod-avatar-focus-indicator-gap, var(--spectrum-avatar-focus-indicator-gap)) * 2));
+  
     border-style: solid;
-    border-radius: var(--spectrum-avatar-border-radius);
+    border-width: var(--mod-avatar-focus-indicator-thickness, var(--spectrum-avatar-focus-indicator-thickness));
+    border-color: var(--highcontrast-avatar-focus-indicator-color, var(--mod-avatar-focus-indicator-color, var(--spectrum-avatar-focus-indicator-color)));
+    border-radius: var(--mod-avatar-border-radius, var(--spectrum-avatar-border-radius));
   }
 }
+
+.spectrum-Avatar-link {
+  outline-color: transparent;
+  outline-style: solid;
+  outline: none;
+}
+
 .spectrum-Avatar-image {
-  inline-size: var(--spectrum-avatar-width);
-  block-size: var(--spectrum-avatar-height);
+  inline-size: var(--mod-avatar-inline-size, var(--spectrum-avatar-inline-size));
+  block-size: var(--mod-avatar-block-size, var(--spectrum-avatar-block-size));
+
+  border-radius: var(--mod-avatar-border-radius, var(--spectrum-avatar-border-radius));
```

Comment:
_💭 Thought, not a necessary change:_

We could also achieve this by setting 

`boder-radius: inherit;`

on both .spectrum-Avatar-image and .spectrum-Avatar-link

Reply:
Interesting suggestion. I looked into this briefly, and I think we _might_ not be able to safely do this and have it work like we want it to: https://css-tricks.com/shadow-roots-and-inheritance/